internal/ignition/ignition.go

```diff
@@ -591,6 +597,41 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte,
 	return nil
 }
 
+func (g *installerGenerator) importClusterTLSCerts() error {
+	tlsDir := os.Getenv("ASSISTED_SERVICE_CLUSTER_TLS_CERTS_DIR")
```

Comment:
to say the truth i prefer moving it to main Options and provide it as param to NewGenerator.

Reply:
See example here
https://github.com/openshift/assisted-service/blob/15cca0fae0e3cc52cb8b083c0b2775a7b5ec5e87/cmd/main.go#L143